File: AspNetMvcUnleashed/0672329980_source/Chapter07/VB/MvcApplication1/MvcApplication1/Controllers/OrderController.vb
Public Class OrderController
    Inherits System.Web.Mvc.Controller

    ' GET: /Order/Create
    Function Create() As ActionResult
        Return View()
    End Function

    ' POST: /Order/Create
    <AcceptVerbs(HttpVerbs.Post)> _
    Function Create(ByVal shipping As Address, ByVal billing As Address) As ActionResult
        ' Insert into database
        Return RedirectToAction("Index")
    End Function

 End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter07/CS/MvcApplication1/MvcApplication1/Controllers/OrderController.cs
using System.Web.Mvc;
using MvcApplication1.Models;

namespace MvcApplication1.Controllers
{
    public class OrderController : Controller
    {

        //
        // GET: /Order/Create

        public ActionResult Create()
        {
            return View();
        } 

        //
        // POST: /Order/Create

        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult Create(Address shipping, Address billing)
        {
            // Insert into database
            return RedirectToAction("Index");
        }


    }
}
